mesh/mesh.hpp

```diff
@@ -1344,6 +1369,59 @@ class GeometricFactors
    Vector detJ;
 };
 
+/** @brief Structure for storing mesh geometric factors: coordinates, Jacobians,
+    and determinants of the Jacobians. */
```

Comment:
"mesh geometric factors" -> "face geometric factors"
"coordinates, Jacobians, and determinants of the Jacobians" -> "coordinates, Jacobians, determinants of the Jacobians, and normal vectors"